pkg/chartutil/create.go

```diff
@@ -299,6 +337,50 @@ spec:
 {{- end }}
 `
 
+const defaultHttpRoute = `{{- if .Values.httpRoute.enabled -}}
+{{- $fullName := include "<CHARTNAME>.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if .Capabilities.APIVersions.Has "gateway.networking.k8s.io/v1" -}}
+apiVersion: gateway.networking.k8s.io/v1
+{{- else -}}
+apiVersion: gateway.networking.k8s.io/v1alpha2
```

Comment:
Is v1 common enough now to just drop v1alpha2 from the template?

Reply:
nice spot @kfox111 -- this code is going to be in Helm and created charts for a "long time". I think if v1 is already available, then don't include support for the alpha version. A user who really needs this can manually edit.